<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings_</class>
 <widget class="QWidget" name="Settings_">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="OnlineBookmarks_">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Account_">
      <attribute name="title">
       <string>Accounts</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="HorizontalLayout_">
         <item>
          <widget class="QLabel" name="Label1_">
           <property name="text">
            <string>Bookmarks service</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Services_">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="HorizontalSpacer_">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="MainLayout_">
         <item>
          <widget class="QTreeView" name="AccountsView_"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="ControlLayout_">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QPushButton" name="Add_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Edit_">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Edit</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Delete_">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="VerticalSpacer_">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="LoginFrame_">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QGroupBox" name="AuthGroup_">
                <property name="title">
                 <string/>
                </property>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Login</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLineEdit" name="Login_"/>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Password</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLineEdit" name="Password_">
                   <property name="echoMode">
                    <enum>QLineEdit::PasswordEchoOnEdit</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QPushButton" name="Apply_">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Apply</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SyncSettings_">
      <attribute name="title">
       <string>Sync settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="Label_">
         <property name="text">
          <string>Select active services</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QListView" name="ActiveServices_"/>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="ConfirmSend_">
         <property name="text">
          <string>Confirm sending bookmarks on addition</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <layout class="QGridLayout" name="SyncOptionsLayout_">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="Bookmarks2Services_">
           <property name="text">
            <string>Send bookmarks to active services every</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="Bookmarks2ServicesPeriod_">
           <item>
            <property name="text">
             <string>Never</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>On startup</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Day</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Week</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Month</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Year</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QCheckBox" name="Services2Bookmarks_">
           <property name="text">
            <string>Receive bookmarks from active services every</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="Services2BookmarksPeriod_">
           <item>
            <property name="text">
             <string>Never</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>On startup</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Day</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Week</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Month</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Year</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="ShowServices_">
         <property name="text">
          <string>Ask user about services witch bookmarks send should be</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
